rpcd/playbooks/roles/rpc_maas/tasks/rabbitmq_user.yml

```diff
@@ -51,6 +46,6 @@
     write_priv: ".*"
     tags: "administrator"
     state: "present"
-  with_items: vhost_json
+  with_items: vhost_list.json
```

Comment:
So vhost_list.json is the register returned in a json format? Is there another way of expressing this to make it less confusing, like perhaps ``vhost_list | json`` or ``vhost_list.json()`` ?